benchmarks/media-streaming/client/files/videoperf/httperf.c

```diff
@@ -79,7 +79,7 @@ test.
 #endif
 
 #define RATE_INTERVAL	5.0
-#define STATS_INTERVAL 60.0
+#define STATS_INTERVAL 15.0
```

Comment:
See above comment - why should we make verbose printing the default option?

Reply:
Verbose is removed.